README.md

```diff
@@ -8,7 +8,35 @@ This repository serves as a sample GitOps configuration repository that is meant
 
 Please refer to the Amazon EKS Blueprints Quick Start [Getting Started](https://aws-quickstart.github.io/cdk-eks-blueprints/getting-started/) guide for details on how to bootstrap an EKS cluster with the workload configuration contained in this repository.
 
-## Usage 
+## Usage
+
+### Usage for Multi Repository example
+
+Under the `multi-repo` folder you can find an example of bootstrapping ArgoCD App of Apps application that points to other applications that resides in other git repositories. This way a deployment of every application can be managed independently by the application team in the application's repository. The platform team on the other hand, own this repository (the ArgoCD app of apps configuration) and only need to onboards new applications to the cluster by adding them to the App of Apps chart
+
+To bootstrap an EKS cluster with the configuration under the `multi-repo` folder, you first have to apply the ArgoCD [Projects](https://argo-cd.readthedocs.io/en/stable/user-guide/projects/) manifests resides in the [argo-app-projects](./multi-repo/argo-app-projects/) folder. These Projects are used by the ArgoCD Applicaitons defined in the [argo-app-of-apps](./multi-repo/argo-app-of-apps/) folder. The projects are used in order to restrict the deployment of an application to a specific target namespace. This way we ensure that even if the application team try to deploy their application to a different namespace (by adding explicit namespace definition to their Kubernetes manifests), ArgoCD will not apply the application manifests because of that restriction. Applying the manifests is done by:
+
+```bash
+kubectl apply -f multi-repo/argo-app-projects/
```

Comment:
IIRC, we discussed applying projects through the app of apps approach? Did it not work?
E.g. putting the projects under `argo-app-project` and marking them as wave 0 (app-projects become yet another app of apps). The rest of the manifests could be wave1. 

If it is supported, let's call out that these steps are fow the manual bootstrap (and add a link to the turn key pattern).

Reply:
We tried to apply projects as part of the `teamManifestDir` in the Team construct, but it didn't work as the apply happens before ArgoCD addon is being deployed.  
Therefore we defined this projects in the ArgoCD addon creation, and actually this is quite useful as we can generate all of these projects dynamically (I will share this in the pattern repository).